.github/workflows/ef-tests.yml

```diff
@@ -0,0 +1,79 @@
+name: EF Tests
+
+on:
+  pull_request:
+    branches:
+      - main
+
+permissions: read-all
+
+jobs:
+  ef-tests:
+    # trunk-ignore(actionlint/runner-label)
```

Comment:
you need to add a file `.github/actionlint.yaml` with

```
self-hosted-runner:
  labels:
    - ubuntu-latest-16-cores
```

instead

Reply:
resolved.